mails/themes/modern/core/download_product.html.twig

```diff
@@ -259,7 +259,14 @@
                                     </tr>
                                     <tr>
                                       <td align="left" style="font-size:0px;padding:10px 25px;word-break:break-word;">
-                                        <div style="font-family:Open sans, arial, sans-serif;font-size:14px;line-height:25px;text-align:left;color:#363A41;"> {virtualProducts} </div>
+                                        <div style="font-family:Open sans, arial, sans-serif;font-size:14px;line-height:25px;text-align:left;color:#363A41;">
+                                          {% if templateType == 'html' %}
+{virtualProducts}
+{% endif %}
+                                          {% if templateType == 'txt' %}
```

Comment:
same here

Reply:
Oh, just noticed these files are generated -_-

Reply:
This one is generated by the [mjml-theme-converter](https://github.com/PrestaShop/mjml-theme-converter), I can't change it